src/main/java/uk/gov/companieshouse/model/validator/EndDateValidator.java

```diff
@@ -26,9 +26,13 @@ public String validateEndDate(Appeal appeal) {
         if (!continued && rawEndDate == null) {
             return EMPTY_END_DATE;
         }
-        if (!rawEndDate.isEmpty() && continued) {
+
+        if (rawEndDate != null  && continued) {
             return END_DATE_CONTINUED_TRUE;
         }
+        if (rawEndDate == null) {
```

Comment:
What's this extra check for? Want to understand what the first two if statements miss

Reply:
This is to catch the mention that Jamie said of if continued is true and End date is null. Weirdly as it currently stands if that comes through then we get a null pointer so this will check if end date is null AFTER confirming that continued is true and pass the null string.

Reply:
It would be enough to add an annotation @NotBlank(message = "illnessEnd must not be blank") on `illnessEnd` inside the IllnessReason model?